app/controllers/api/v1/rents_controller.rb

```diff
@@ -4,23 +4,20 @@ class RentsController < ApiController
       before_action :set_locale
 
       def create
-        new_rent = current_user.rents.build
-        new_rent.assign_attributes(rents_params)
-
-        authorize new_rent
-
-        if new_rent.save
-          RentMailer.new_rent_notification(new_rent).deliver_later
+        rent = Rent.new(rents_params)
+        authorize rent
+        rents = CreateRent.call(rents_params)
```

Comment:
Use `rents = CreateRent.call(params: rents_params)` this way you can use the params inside the Interactor as a hash.